client/api/src/lib.rs

```diff
@@ -18,20 +18,17 @@
 #![warn(missing_docs)]
 
 pub mod backend;
-pub mod blockchain;
 pub mod call_executor;
 pub mod client;
-pub mod error;
 pub mod execution_extensions;
 pub mod light;
 pub mod notifications;
 
-// TODO: avoid re-exports
+pub use sp_blockchain as blockchain;
```

Comment:
What are we going to do with this crate? It seems that now we either import `sp_blockchain` directly or use `client_api::blockchain`. Would be good to have some consistency.

Also I'm still not in favor of re-exports in that crate, I think it makes more sense to mention the module name on the import path.

Reply:
this is mostly for convenience for me for now. I consider it part of the unification to remove this and other lines in here in favor of explicit crate names.